Networking/Networking/Model/PaymentGateway.swift

```diff
@@ -0,0 +1,108 @@
+import Foundation
+
+/// Represents a Payment Gateway.
+///
+public struct PaymentGateway: Equatable {
+
+    /// Features for payment gateway.
+    ///
+    public enum Features: Equatable {
+        case products
+        case refunds
+        case custom(raw: String)
+    }
+
+    /// Site identifier.
+    ///
+    public let siteID: Int64
+
+    /// Gateway Identifier.
+    ///
+    public let gatewayID: String
+
+    /// Title for the payment gateway.
+    ///
+    public let title: String
+
+    /// Description of the payment gateway.
+    ///
+    public let description: String
+
+    /// Wether the payment gateway is enabled on the site or not.
+    ///
+    public let enabled: Bool
+
+    /// List of features the payment gateway supports.
+    ///
+    public let features: [Features]
+}
+
+// MARK: Gateway Decodable
+extension PaymentGateway: Decodable {
```

Comment:
I like that `Decodable` implementation is separated from the properties with `Equatable` implementation above 👍